src/software/ai/hl/stp/stp_test.cpp

```diff
@@ -14,47 +15,52 @@
 class STPTest : public ::testing::Test
 {
    public:
-    STPTest() : stp([]() { return nullptr; }, DynamicParameters->getAiControlConfig(), 0)
-    {
-    }
+    STPTest() : stp([]() { return nullptr; }, ai_control_config, 0) {}
```

Comment:
this is passing in an uninitialized `ai_control_config`, which I think is causing a segfault for me locally. `ai_control_config` should be initialized in the constructor

Reply:
very strange, doesn't seem to segfault for me but it def should

fixed!